### ISSUE TITLE
The loss of referring segmentation

### ISSUE THREAD

**Issue by jshilong** [microsoft/X-Decoder#8]
Thanks for the great work,

In section 4.1, you mentioned that the model was pre-trained on "panoramic segmentation, image-text"
pairs (itp), and referring segmentation.` I can't find the details of how you use `Referring Segmentation` data in 3.4, would you mind providing more details about `Referring Segmentation` data loss in the pre-training phase? thanks

**Comment by yxchng** [comment]
@MaureenZOU 
1.Just to clarify. Does "refcoco(g/+)" mean only refcoco+ and refcocog, with refcoco excluded?
2. What does this mean "In addition, those images that do not have referring seg ground truth, we use instance segmentation as labels (e.g. person -> all person instance)"?
